dlpy/transformers/bert_model.py

```diff
@@ -222,54 +320,75 @@ def _load_embedding_table(self):
                 
             position_embedding_table = self._base_model.state_dict()[param_tensor].numpy()
                         
```

Comment:
The code you referenced is actually pulling from the PyTorch state dictionary.  I don't think you are suggesting that we document the PyTorch dictionary, but rather the embedding table we create for BERT.  I'm not sure how much value we derive from more documentation here because the format of the table is very much tied to the CAS implementation of embedding.  An interested user is not going to glean much information from it.